cli/cmd/deploy/deploy.go

```diff
@@ -65,6 +65,7 @@ func DeployCmd(ch *cmdutil.Helper) *cobra.Command {
 	deployCmd.Flags().StringVar(&opts.ProdVersion, "prod-version", "latest", "Rill version (default: the latest release version)")
 	deployCmd.Flags().StringVar(&opts.ProdBranch, "prod-branch", "", "Git branch to deploy from (default: the default Git branch)")
 	deployCmd.Flags().IntVar(&opts.Slots, "prod-slots", local.DefaultProdSlots(ch), "Slots to allocate for production deployments")
+	deployCmd.Flags().BoolVar(&opts.UploadVariables, "upload-variables", true, "Upload local .env file to Rill Cloud")
```

Comment:
Can we call this `--push-env` instead? To align with the terminology of the `rill env push` command